src/Loggers/LogEmbeddings.jl

```diff
@@ -65,7 +65,9 @@ function write_sprite(img_labels::AbstractArray, matrix_path::AbstractString)
     arranged_augment_square_CHW = zeros((3, sprite_size, sprite_size))
     arranged_augment_square_CHW[:, 1:size(arranged_img_CHW, 2), :] = arranged_img_CHW
     sprite_path = joinpath(matrix_path, "sprite.png")
-    save(sprite_path, colorview(RGB, arranged_augment_square_CHW))
+    open(sprite_path; write=true) do io
```

Comment:
It is annoying the this is needed 
Should an issue  be openned in FileIO.jl?

Reply:
So there's two issues here; one is easy (https://github.com/JuliaCloud/AWSS3.jl/issues/174), the other is that ImageMagick types paths to `AbstractString` which is too strict (FilePathsBase PosixPaths for example are probably OK too), but it isn't fully generic because it eventually goes to
```
ccall((:MagickWriteImages, libwand), Cint, (Ptr{Cvoid}, Ptr{UInt8}, Cint), wand, filename, true)
```
in `writeimage` and I don't think that can handle s3 paths. Possibly ImageMagick should add another method for `writeimage` where unknown filename types are handled by opening a stream like we do here?